### PR TITLE
Add Windows runner configuration

### PR DESCRIPTION
**Pull request by FriedrichFroebel**
This adds Windows support for the Travis tests (see #134).

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,11 +2,19 @@ language: python
...
+    - name: "Python 3.7 on Windows"
+      os: windows
+      language: shell  # `language: python` is not supported for now.
+      before_install:
+        - choco install python --version 3.7.7
+        - python -m pip install --upgrade pip
+      env: PATH=/c/Python37:/c/Python37/Scripts:$PATH
 
 install:
   - python3 setup.py build install
```
Comment (tomasbedrich) [inline .travis.yml:20]: I think on Windows the `python3` command is not present by default, so this must be overridden.

Comment (FriedrichFroebel) [reply .travis.yml:26]: I am currently working on it and already replaced these. Let's see if this is enough.

**Comment by FriedrichFroebel** [comment]
I fixed the coverage report with `coveralls` as well, but I am not sure about the following message:

> PyYAML is not installed, skipping .coveralls.yml.

@tomasbedrich As I am not sure about it: Should we install `pyyaml` as well to eliminate this message and therefore allow handling the `.coverall.yml` of the repository as well?

**Comment by tomasbedrich** [comment]
[According to docs](https://coveralls-python.readthedocs.io/en/latest/usage/configuration.html#configuration):

> If coveralls-python is being run on TravisCI, it will automatically set the token for communication with coveralls.io. Otherwise, you should set the environment variable COVERALLS_REPO_TOKEN, ...

So I think we are safe to delete `.coveralls.yml`.

**Pull request merged**